### DISCUSSION TITLE
How to add page title?

### DISCUSSION THREAD

**Discussion by Sarvom** [fastn-stack/fastn#813] [Q&A] [answered]
How to add title and favicon for Website using ftd?

**Comment by amitu** [comment, marked as answer]
You can use `ftd.document` as the top level node in your page to set page title. Use the latest release, and try this in a document:

```ftd
-- ftd.document: My title
-- ftd.text: this page has a title
-- end: ftd.document
```

Ensure that `ftd.document` is the top level element, you can not put a `ftd.document` in another container like `ftd.row`, `ftd.column` or even `ftd.document` etc. `title` argument in `ftd.document` has type `caption`.

To add `favicon` you have to add it to `fastn.package` declaration in your `FASTN.ftd` file:

```
-- import: fastn
-- fastn.package: hello
favicon: /static/favicon.png
```

**Reply by Sarvom** [reply]
![image](https://user-images.githubusercontent.com/45617260/229692523-32146585-db68-4bf6-b3a5-9b885ae7e74a.png)

This error is occurring while try to implement ftd.document

**Reply by amitu** [reply]
Error says "ftd.document can't have siblings".  Looks like `ftd.document` is not the top level element. It must be the top level element, with no siblings. Use `ftd.document` as the outer most container in your document and put everything inside it. No UI must come after `-- end: ftd.document` (variable, component definitions are okay to come after `-- end: ftd.document`).
